portfolio/src/main/webapp/task.js

```diff
@@ -61,19 +63,19 @@ function createNewTask() {
 
   const newTask = new Task(
       name, description, getDurationMinutes(length, lengthUnit),
```

Comment:
nit: Can we either put these parameters all on the same line, or give them each their own line?

Reply:
Done.